### ISSUE TITLE
Missing outline in High Contrast Mode for video in Welcome page.

### ISSUE THREAD

**Issue by peterdobal** [carbon-design-system/carbon-for-ibm-dotcom#6322]
## Detailed description

> Describe in detail the issue you're having.

Setup needed for HCM:
Open IBM Docs in FF
Hit Shift+Alt+PrtSc and start High Contrast Mode
After question if you want to enter to HCM wait few sec and everything on view should be change to HCM(depends on thinkpad older WS did not offer question and it took 1-3 seconds to load)
In some cases refresh of page is required.

> Is this a feature request (new component, new icon), a bug, or a general
> issue?
BUG
> Is this issue related to a specific component?

> What did you expect to happen? What happened instead? What would you like to
> see changed?

> What browser are you working in?
FF 89.0, Windows 10

> What version of Carbon for IBM.com are you using?

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

## Steps to reproduce the issue

1. Open https://ibmdocs-dev.mybluemix.net/docs/en
2. Navigate with tab to  video
3..... 

Actual  behavior:
Currently video  is excluded  and  user  is not able to  go through  it because outline is missing in HCM.

Expected behavior:
When user work in HCM and navigate with tab, expectation is  video on  welcome page is part of  whole navigation and user is able  to  navigate and run in  properly.

## Additional information

- https://accessabilly.com/accessibility-issues-concerning-windows-high-contrast-mode/
- https://www.maxability.co.in/2021/06/13/windows-high-contrast-mode-and-accessibility-testing/
- https://johnsweetaccessibility.com/2020/06/focus-style-strategies/

**Comment by ljcarot** [comment]
@peterdobal It would be most helpful if you can provide screenshot from the Accessibility Checker where this issue has been flagged as well if the checker says it's a violation, a needs review, or a recommendation.

**Comment by peterdobal** [comment]
Wer talking about main video  on Home page or welcome page and also about different  accessibility tool or approach then IBM Accessibility Checker.

**Comment by peterdobal** [comment]
![video section](https://user-images.githubusercontent.com/15247068/121355392-20276d00-c930-11eb-9a26-411dceba9cf4.png)

**Comment by peterdobal** [comment]
![video HCM](https://user-images.githubusercontent.com/15247068/121523741-d6ee2080-c9f6-11eb-9f64-c15c76848fb4.png)

**Comment by RobertaJHahn** [comment]
@Praveen-sr Can you have Ram look at this issue to determine the severity.  Thanks.

**Comment by RobertaJHahn** [comment]
Spoke to Ram in our Prod QA meeting this morning.  He said that WCAG does not require High Contrast Mode (HCM) testing.  He will talk to his testing team to find out what additional % time would be required if the HCM testing would be added to scope.  

The priority of this defect would be High if this fix is going to be done.

**Comment by RobertaJHahn** [comment]
If this HCM testing was added for all components, the accessibility testing would increase by about 15 %.

**Comment by ljcarot** [comment]
@peterdobal Due to our current accessibility budget, we have not included HCM testing this year as we've been focused on WCAG 2.1 level AA.  As a result, we will move the HCM issues to Backlog until next year.

**Comment by Stefan8412** [comment]
Hello , can you please give us update regards above mentioned issue, thanks
